Android/app/src/main/java/io/github/project_travel_mate/utilities/WorldClockActivity.java

```diff
@@ -118,19 +97,40 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
-
-    /**
-     * Pulls 12/24 mode from system settings
-     */
-    private boolean get24HourMode() {
-        return DateFormat.is24HourFormat(mContext);
+    private void getGMTTime() {
+        mCurrent = Calendar.getInstance();
+        mCurrent.add(Calendar.HOUR, -2);
+        /* Time calculation according to time zone */
+        mMiliSeconds = mCurrent.getTimeInMillis();
+        TimeZone tzCurrent = mCurrent.getTimeZone();
+        int offset = tzCurrent.getRawOffset();
+        if (tzCurrent.inDaylightTime(new Date())) {
+            offset = offset + tzCurrent.getDSTSavings();
+        }
+        mMiliSeconds = mMiliSeconds - offset;
     }
 
-    private void setFormat() {
-        if (get24HourMode()) {
-            mDigitalClock.setFormat24Hour(mHours24);
-        } else {
-            mDigitalClock.setTimeZone(mHours12);
-        }
+    private String getSavedValue(String key) {
```

Comment:
no need of a new function just for this purpose

Reply:
Shall I remove it?

Reply:
yes